Automattic/jetpack
# External Media: support editor changes in WordPress 6.5

Fixes #36186

## Proposed changes:

Add a new conditional to ensure the External Media feature continues to work with WordPress 6.5.

This should also avoid warnings like this one:

```
select( 'core/edit-post' ).isInserterOpened is deprecated since version 6.5. Please use select( 'core/editor' ).isInserterOpened instead. [index.js:77:9](wp-content/plugins/gutenberg/packages/deprecated/src/index.js)
```

### Other information:


## Jetpack product discussion

* No

## Does this pull request change what data or activity we track or use?
* No

## Testing instructions:

* Install the WordPress Beta Tester plugin
* Go to Posts > Add New
* Add a new image block
* Pick one of the external media sources like Pexels
    * You should be able to pick an image from there.
* Go to Tools > WordPress Beta Tester
* Switch to Bleeding edge
* Go to Dashboard > updates and update to a bleeding edge version of WordPress
* Repeat your external media test to ensure the feature continues to work with WordPress 6.5.
* Ensure you do not see the warning above when opening the browser's JavaScript console.
